### PR TITLE
Updated Events with new Graphics

### PR DESCRIPTION
**Pull request by Mustaballer**
Currently missing graphic for React Workshop with updated date.

### CODE REVIEW

**Review by YuYing-Liang — CHANGES_REQUESTED** [review]
Could you put all the events on the slideshow to be in chronological order from left to right
Could you put the deadline for FYA recruitment (Sept. 19th @ 11:59PM)
Unrelated: could you change the email address in the footer from uoft@ieee.org to general@ieee.utoronto.ca

**Review by Mustaballer — COMMENTED** [review]
For the React Graphic, I'm still waiting for Marketing team to give me an updated graphic so for now I will focus on the other changes

**Comment by YuYing-Liang** [comment]
@Mustaballer looks good, could you change "Our Sponsors" to "Our Past Sponsors"

File: src/Components/Home/Sponsors/Sponsors.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const Sponsors = () => {
 
     return (
         <div id="sponsors" className={styles.sponsorsDiv}>
-            <h2 className={styles.sponsorsDivHeading}>Our Sponsors</h2>
+            <h2 className={styles.sponsorsDivHeading}>Past Sponsors</h2>
```
Comment (YuYing-Liang) [inline src/Components/Home/Sponsors/Sponsors.js:38]: Could this be "Our Past Sponsors"

**Review by YuYing-Liang — APPROVED** [review]

**Pull request merged**